composer.json

```diff
@@ -6,6 +6,9 @@
     "autoload": {
         "psr-4": { "": "src/" }
     },
+    "autoload-dev": {
+        "psr-4": { "Tests\\": "tests/" }
```

Comment:
Why not use `AppBundle\\Tests\\` here so that we can keep using the `AppBundle\Tests` namespace prefix in the test classes?